espflash/src/flasher/mod.rs

```diff
@@ -93,49 +107,89 @@ pub enum FlashMode {
 #[non_exhaustive]
 #[repr(u8)]
 #[strum(serialize_all = "SCREAMING_SNAKE_CASE")]
+#[doc(alias("esp_image_flash_size_t"))]
 pub enum FlashSize {
     /// 256 KB
-    _256Kb = 0x12,
+    _256Kb,
     /// 512 KB
-    _512Kb = 0x13,
+    _512Kb,
     /// 1 MB
-    _1Mb = 0x14,
+    _1Mb,
     /// 2 MB
-    _2Mb = 0x15,
+    _2Mb,
     /// 4 MB
     #[default]
-    _4Mb = 0x16,
+    _4Mb,
     /// 8 MB
-    _8Mb = 0x17,
+    _8Mb,
     /// 16 MB
-    _16Mb = 0x18,
+    _16Mb,
     /// 32 MB
-    _32Mb = 0x19,
+    _32Mb,
     /// 64 MB
-    _64Mb = 0x1a,
+    _64Mb,
     /// 128 MB
-    _128Mb = 0x21,
+    _128Mb,
+    /// 256 MB
+    _256Mb,
 }
 
 impl FlashSize {
-    fn from(value: u8) -> Result<FlashSize, Error> {
+    // components/esptool_py/esptool/docs/en/advanced-topics/firmware-image-format.rst:19
```

Comment:
```suggestion
    // https://docs.espressif.com/projects/esptool/en/latest/esp8266/advanced-topics/firmware-image-format.html?highlight=0xe9#file-header 
```
I think its better to point at the rendered documentation when possible, this way, if the line or file name changes, it may still be valid

Reply:
Such a nice feature with `/esp-MODEL/`!